tasks/src/main/scala/dagr/tasks/bwa/BwaBacktrack.scala

```diff
@@ -16,12 +16,17 @@ import scala.collection.mutable.ListBuffer
   * BAM file via bwa backtrack/bwa short alignment.
   */
 class BwaBacktrack(val unmappedBam : PathToBam, val mappedBam : PathToBam,
-                   val ref : PathToFasta, val tmpDir: Option[DirPath] = None
+                   val ref : PathToFasta, val tmpDir: Option[DirPath] = None,
+                   val maxDiffOrMissingProb: Option[Either[Int, Float]] = None,
```

Comment:
It's really annoying that scala's numeric types don't have a common superclass, so you can't just do `Option[Number]`